### DISCUSSION TITLE
Does it work with MDX stories?

### DISCUSSION THREAD

**Discussion by morewings** [storybookjs/addon-designs#132] [Q&A] [answered]
Does it work with MDX stories? Can I see an example code?

**Comment by pocka** [comment, marked as answer]
Yes, [by specifying `parameters` prop to `<Story>` and passing `withDesign` decorator to `<Meta>`](https://github.com/pocka/storybook-addon-designs/issues/22#issuecomment-544268494).

**Reply by morewings** [reply]
Thanks
